website/docs/r/lb_vpx.html.markdown

```diff
@@ -1,63 +1,62 @@
-* `private_vlan_id` - (Optional, integer) Private VLAN ID that is used for the private network interface of the VPX load balancer. Accepted values can be found [in the VLAN docs](https://control.softlayer.com/network/vlans). Click  the desired VLAN and note the ID in the resulting URL. Or, you can also [refer to a VLAN by name using a data source](../d/network_vlan.html).
-* `public_subnet` - (Optional, string) Public subnet that is used for the public network interface of the VPX load balancer. Accepted values are primary public networks and can be found [in the subnet docs](https://control.softlayer.com/network/subnets).
-* `private_subnet` - (Optional, string) Public subnet that is used for the private network interface of the VPX load balancer. Accepted values are primary private networks and can be found [in the subnet docs](https://control.softlayer.com/network/subnets).
-* `tags` - (Optional, array of strings) Set tags on the VPX load balancer instance.
-
-**NOTE**: `Tags` are managed locally and not stored on the IBM Cloud service endpoint at this moment.
-
-## Attributes Reference
-
-The following attributes are exported:
-
-* `id` - The internal identifier of a VPX load balancer
-* `name` - The internal name of a VPX load balancer.
-* `vip_pool` - List of virtual IP addresses for the VPX load balancer.
-* `management_ip_address` - Private address of VPX UI
+---
+layout: "ibm"
+page_title: "IBM: lb_vpx"
+sidebar_current: "docs-ibm-resource-lb-vpx"
+description: |-
+  Manages IBM VPX Load Balancer.
+---
+
+# ibm\_lb_vpx
+
+Provides a resource for VPX load balancers. This allows VPX load balancers to be created, updated, and deleted.
+
+**NOTE**: IBM VPX load balancers consist of Citrix NetScaler VPX devices (virtual), which are currently priced on a per-month basis. Use caution when creating the resource because the cost for an entire month is incurred immediately upon creation. For more information about pricing, see the [network appliance docs](http://www.softlayer.com/network-appliances). Under the Citrix log, click **See more pricing** for a current price matrix.
+
+You can also use the following REST URL to get a listing of VPX choices along with version numbers, speed, and plan type:
+
+```
+https://<userName>:<apiKey>@api.softlayer.com/rest/v3/SoftLayer_Product_Package/192/getItems.json?objectMask=id;capacity;description;units;keyName;prices.id;prices.categories.id;prices.categories.name
+```
+
+## Example Usage
+
+Review the [Bluemix Infrastructure (SoftLayer) docs]((http://sldn.softlayer.com/reference/datatypes/SoftLayer_Network_Application_Delivery_Controller) for more information.

```

Comment:
What information would be helpful to users this link? Takes you to the Network_Application_Delivery_Controller data source reference. I want to tell them what they can look for in there. @vkalangu